webpack.config.js

```diff
@@ -99,6 +99,27 @@ customEntryPoints['wizard/wizard'] = exposeEntry('tec.tickets.wizard', __dirname
  */
 doNotPrefixSVGIdsClasses(defaultConfig);
 
+defaultConfig.externals = [
```

Comment:
I'm curious: why are the externals added this way, instead of to the `module.exports` object below?